### ISSUE TITLE
When will the updated google auth stuff in the dev branch be part of an official release?

### ISSUE THREAD

**Issue by klumsy** [aspnet/AspNetKatana#258]
Given google retiring their googleplus oauth api endpoint starting Jan 28th through March, and the fact the the dev branch already has a fix to use their maintained and newer oauth endpoints, when will this be rolled into an official release. We have changed over to the dev branch, but would like to use go back to using the official release nuget package.

**Comment by Tratcher** [comment]
As soon as we get some build issues worked out and run some sanity tests on the final output.

**Comment by klumsy** [comment]
Thanks. So i take it then that its more on the order of days/weeks.. rather than months/years? I'm only asking for planning purposes, and because i couldn't be sure of the plans based on looking at official release history which ranges from a number of times a year, to a year plus. I am do not want to put pressure on rushing something that isn't ready but rather just planning for the future. Thanks. We are very glad that this change was added to the dev branch. After we got the notifications from google, and looked at a number of options, the best course of action we came up with was take this library and fork it locally and make the same sort of changes, and then a few days later we saw those changes show up in the dev branch, and we are much happier to go the route of following a supported official release than maintaining our own.
